cmd/podman/build.go

```diff
@@ -184,7 +184,7 @@ func buildCmd(c *cli.Context) error {
 	if err != nil {
 		return errors.Wrapf(err, "error building system context")
 	}
-
+	systemContext.AuthFilePath = getAuthFile(c.String("authfile"))
```

Comment:
nit of the tiniest nits.  If possible, it would be nice to have the blank line above this still.  But only adjust if you need other refinements for this PR.